packages/react-inbox/src/hooks/index.ts

```diff
@@ -1 +1,2 @@
 export { default as useInbox } from "./use-inbox";
+export { default as useClickOutside } from "./use-click-outside";
```

Comment:
why not just named export `useClickOutside` instead of aliasing default export?

Reply:
idk, no reason